### PR TITLE
View

### PR DESCRIPTION
**Pull request by tonnytech**
### Did the following in this pull request:

- Implement the design from the sneak peek wireframes.
- Use methods that I have created in the models.

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,32 @@
...
+                <div>
+                    <span>Comments:<%= post.comments_counter || 0 %> </span> 
+                    <span>Likes:<%= post.likes_counter || 0 %></span>
+                </div>    
+            </div>
+        <% end %>
+        <button type="button" class="see-posts">See all posts</button>
+        <% else %>
+            <p>No posts found.</p>
+        <% end %>
```
Comment (elisha2kyakpo1) [inline app/views/users/show.html.erb:31]: - [ ] Kindly add basic CSS to the user show page to match the wireframe design. Your design is different from the wireframe.
- [ ] The "See all Posts" button doesn't work, I had to type the URL in the browser to see all the posts, Kindly fix this issue for an easy user experience.

user-show

design

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,33 @@
...
+            <% if post.recent_comments.present? %>
+                <div class="">
+                        <% post.recent_comments.each do |comment| %>
+                            <div class="">
+                                <div class=''> 
+                                    <%= @user.name%> : <span><%= comment.text %></span>
+                                </div>
+                            </div>
+                        <% end %>
+                </div>
```
Comment (elisha2kyakpo1) [inline app/views/posts/index.html.erb:25]: - [ ] Kindly make this page design to match the wireframe, observe the difference, and make changes.
- [ ] Note that this page should have a pagination button.

posts


wireframe

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @tonnytech, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description are good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.
👏 The app works as expected without errors, though some features are missing.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @tonnytech   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter error :heavy_check_mark: 
- You implemented the suggested changes :heavy_check_mark: 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**